modular_skyrat/modules/aesthetics/morgue/code/morgue.dm

```diff
@@ -1,6 +1,2 @@
 /obj/structure/bodycontainer
 	icon = 'modular_skyrat/modules/aesthetics/morgue/icons/morgue.dmi'
```

Comment:
There's a newline here according to the files I pushed, if somehow its not here I'm gonna cry